### ISSUE TITLE
Backdrops enabled. Backdrops enabled in parts. 

### ISSUE THREAD

**Issue by MakD** [krlsantcard/jellyfin-themes#1]
After enabling the theme, and enabling backdrops, whenever I open any movie page, the backdrop forst loads from the top and micro second later the bottom part loads. @krlsantcard

**Comment by krlsantcard** [comment]
Yes that's right, in fact something to work, just not time enough. This weekend I will try, and let u know. I just make those chances just for fun, cus I don't use jellyfin in clients just for server.
